Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -9,10 +9,26 @@
 import UIKit
 
 class PersonCell: UICollectionViewCell {
+    static let personIdentifier = "personCell"
 
+    @IBOutlet var personPhotoImageView: UIImageView!
+    @IBOutlet var nameLabel: UILabel!
+    @IBOutlet var dateLabel: UILabel!
```

Comment:
Since changes are already going to be made, i agree that these should be made to private. We will be refactoring the cells to be programmatic in the future though